eng/ci/official-build.yml

```diff
@@ -0,0 +1,50 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - dev
+      - release/*
+
+# CI only, does not trigger on PRs.
+pr: none
+
+schedules:
+  - cron: '0 0 * * MON'
+    displayName: At 12:00 AM, only on Monday
+    branches:
+      include:
+        - dev
```

Comment:
Do we need to run official builds on a schedule or only on a trigger when something is merged to dev?

Reply:
Official builds need an at-least-weekly schedule for SDL reporting